file_formats/ie_formats/ini.htm

```diff
@@ -1,89 +1,408 @@
+        <br />
+
+        <strong>death_faction = boolean_value</strong><br />
+        <span class="unknown">It is likely that this value controls whether the faction death counter is incremented when the spawned creature is killed.</span> It is only supported by PST and PSTEE.<br />
+        <br />
+
+        <strong>death_team = boolean_value</strong><br />
+        <span class="unknown">It is likely that this value controls whether the team death counter is incremented when the spawned creature is killed.</span> It is only supported by PST and PSTEE.<br />
+        <br />
+
+        <strong>spawn_point = [x.y:dir],[x.y:dir],...</strong><br />
+        This attribute expects one or more map coordinates for the creature. Values can be separated by comma or directly appended without a separator. The engine uses a point randomly if multiple locations are defined. The behavior can be further refined by additional attributes described below. In PST orientation is sometimes omitted and defined by the "facing" attribute instead.<br />
+        <br />
+
+        <strong>point_select = letter</strong><br />
+        This attribute expects a single letter which controls various aspects of the spawning process.<br />
+        Known attribute values are:
+        <ul>
+          <li><span class="code">E</span>: Indicates that the spawn location and orientation are retrieved from predefined game variables defined by "spawn_point_global" and "spawn_facing_global" respectively.</li>
+          <li><span class="code">R</span>: Indicates that the spawn location is chosen at random from the map coordinates defined by the "spawn_point" attribute.</li>
+          <li><span class="code">S</span>: Indicates that a single spawn location is chosen from the map coordinates defined by the "spawn_point" attribute and reused by subsequent spawn operations.</li>
+        </ul>
+
+        <strong>point_select_var = ???</strong><br />
+        <span class="unknown">The meaning of this attribute is unknown.</span><br />
+        <br />
+
+        <strong>facing = number</strong><br />
+        This attribute expects a numeric value in the range 0..15 for the creature's orientation where 0=south, 4=west, 8=north and 12=east. It is used by PST if the orientation is omitted by the "spawn_point" attribute.<br />
+        <br />
+
+        <strong>ignore_can_see = boolean_value</strong><br />
+        This attribute indicates whether a creature can be spawned even if the spawn point is within visual range of the party. By default creatures are only spawned off-screen.<br />
+        <br />
+
+        <strong>check_crowd = boolean_value</strong><br />
+        <span class="unknown">The meaning of this attribute is unknown.</span><br />
+        <br />
+
+        <strong>find_safest_point = boolean_value</strong><br />
+        <span class="unknown">The meaning of this attribute is unknown.</span><br />
+        <br />
```

Comment:
If you want to add guesses like with the other unknowns, mine would be:
- check_crowd: skip the spawn if too many creatures are present in general (think SoD BG areas)
- find_safest_point: consider enemies when placing the spawn. To me this sounds way too computationally expensive for them to have went with the optimal choice, if true. Maybe a heuristic. The simplest would be to just pick the safest point on the list, however, at least the random example I looked at also had point_select=R set.